### PR TITLE
added autoloading

### PR DESCRIPTION
**Pull request by mrvnklm**
Maybe you should update the README

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,14 @@
     },
     "autoload": {
         "psr-4": {
-            "App\\Console\\Commands\\": "src/"
+            "Arkadiusjonczek\\DbCreateCommand\\": "src/"
```
Comment (arkadiusjonczek) [inline composer.json]: Why did you change the namespace? I did this on purpose and it worked very well. ;)

File: composer.json [hunk 2]
```diff
@@ -13,7 +13,14 @@
...
+        }
+    },
+    "extra": {
+        "laravel": {
+            "providers": [
+                "Arkadiusjonczek\\DbCreateCommand\\DbCreateServiceProvider"
```
Comment (arkadiusjonczek) [inline composer.json:22]: I like the idea with the laravel service provider! 👍 🥇

File: src/DbCreateCommand.php [hunk 3]
```diff
@@ -9,7 +9,7 @@
  *
  * @package App\Console\Commands
  */
-class DbCreate extends Command
+class DbCreateCommand extends Command
```
Comment (arkadiusjonczek) [inline src/DbCreateCommand.php:12]: There is no need to use "Command" as postfix. See https://laravel.com/docs/5.8/artisan#command-structure

**Review by arkadiusjonczek — CHANGES_REQUESTED** [review]
Hey @mrvnklm , thank you for your PR. 👍 
I reviewed it and added some comments to review for you. Please take a look at it. :)

File: composer.json (hunk 1 above)
Comment (mrvnklm) [reply composer.json]: Actually I am wondering, why I did that - sorry

File: src/DbCreateCommand.php (hunk 3 above)
Comment (mrvnklm) [reply src/DbCreateCommand.php:12]: True, coming from Symfony made me do this O:-)

Comment (arkadiusjonczek) [reply src/DbCreateCommand.php:12]: I'm fine with this. ;)

File: composer.json (hunk 1 above)
Comment (arkadiusjonczek) [reply composer.json]: Please undo this so I can merge this PR. ;) @mrvnklm

**Review by arkadiusjonczek — CHANGES_REQUESTED** [review]
See my latest comments. ;) @mrvnklm

**Comment by mrvnklm** [comment]
> See my latest comments. ;) @mrvnklm 

I reverted the composer.json

**Comment by arkadiusjonczek** [comment]
Hey @mrvnklm , please update all other namespace to `App\Console\`, too. ;)

**Comment by arkadiusjonczek** [comment]
I added the changes with commit 7ba4d214fddd9a03faab227fe1addd84fa1f11c4
Thank you 🙏🏻

**Pull request closed without merging**